rtv/multireddits.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
```

Comment:
It looks like almost this whole file is copied from **submissions.py**. If possible, I think it would be better to just add a flag to the SubscriptionPage.

```python
page = SubscriptionPage(reddit, term, config, oauth)
```
vs
```python
page = SubscriptionPage(reddit, term, config, oauth, multireddit=True)
```

I don't see a problem with using the same keybindings (``SUBSCRIPTION_SELECT``) and (``SUBSCRIPTION_EXIT``) for both because the pages are functionally the same.